README.md

```diff
@@ -1,8 +1,8 @@
-# IDSeq CLI
+# CZ ID CLI
```

Comment:
Should this be changed to Chan Zuckerberg ID?

Reply:
Yes

Reply:
For context, the README title appears on the cli instructions page as such:

<img width="921" alt="Screen Shot 2021-12-07 at 12 43 58 PM" src="https://user-images.githubusercontent.com/33742286/145103358-df7c74b2-1a4c-4a00-92aa-3e42839ec89d.png">

Reply:
Followed up w/ Olivia. CZ ID is fine for this case.